### DISCUSSION TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'bw'

### DISCUSSION THREAD

**Discussion by gfsd3v** [qutebrowser/qutebrowser#6306] [Q&A] [answered]
Hello guys, I'm having a little problem when trying to execute the bitwarden user script when stating qutebrowser from rofi, when I try to execute the script I get the following error message:

`FileNotFoundError: [Errno 2] No such file or directory: 'bw'`

It seems like the qutebrowser instance initalized from rofi/dmenu does not recognize the bitwarden CLI installed globally using NVM.

When starting qutebrowser via terminal everything work as expected, I'm using qutebrowser-git from AUR, anyone has any idea on what could be causing this problem ?

**Comment by The-Compiler** [comment, marked as answer]
Hey! I wanted to answer in IRC but you were already gone. What does `which bw` say in a terminal? I'd guess it's in `$PATH` in the terminal, but not in graphical applications, because you add it to `$PATH` in something like `.bashrc` instead of `.profile`.

**Reply by gfsd3v** [reply]
Sorry, I losted my internet connection, didn't know if somebody had answered and didn't wanted to ask the same question twice, the `whitch bw` returns `/home/gfs/.nvm/versions/node/v15.12.0/bin/bw`, I didnt't set any path to bw on `.zshrc` or `.profile`, should I add `export PATH="$HOME/.nvm/versions/node/v15.12.0/bin/bw"` to `.profile` ?

**Reply by The-Compiler** [reply]
So the question is why it works in a shell - I assume `echo $PATH` in a shell will show that folder as part of the value. Something must be adding it, as its not there by default.

**Reply by gfsd3v** [reply]
Think I got it, `source /usr/share/nvm/init-nvm.sh` this script from nvm that I have in my `.zshrc` file probably add the nvm bins to the $PATH, I added the same line to my `.profile` and things are working as expected, thank u very much for the help.
